firstvoices/backend/permissions/predicates/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .base import *  # noqa F401
+from .edit2 import *  # noqa F401
```

Comment:
I named it this just to avoid an annoying merge with the categories PR. I'll rename the module to `edit` when i merge them

Reply:
this is renamed now